components/automate-cli/pkg/verifyserver/server/api/v1/certificatevalidation.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber/v2"
+)
+
+func (h *Handler) ValidateCertificate(c *fiber.Ctx) error {
+	req := new(models.CertificateCheckRequest)
+	if err := c.BodyParser(req); err != nil {
+		h.Logger.Error(err.Error())
+		return fiber.NewError(http.StatusBadRequest, "Invalid Body Request")
```

Comment:
The error should be Invalid Request Body

Reply:
done